todo_app/data/session_items.py

```diff
@@ -1,78 +1,171 @@
 from flask import session
+import os
+import requests
+import json
+from todo_app.data.Item import Item
+
+def get_cards():
+    board_id = os.getenv ('BOARD_ID')
+    key = os.getenv ('API_KEY')
+    token = os.getenv ('API_TOKEN')
+    url = f"https://api.trello.com/1/boards/{board_id}/lists"
+
+    headers = {
+       "Accept": "application/json"
+    }
+
+    query = {
+        "key" : key,
+        "token" : token, 
+        "cards" : "open"
+    }
+
+    response = requests.get(
+       url,
+       headers=headers,
+       params = query
+    )
 
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
-
-def get_items():
-    """
-    Fetches all saved items from the session.
-
-    Returns:
-        list: The list of saved items.
-    """
-    return session.get('items', _DEFAULT_ITEMS.copy())
-
-
-def get_item(id):
-    """
-    Fetches the saved item with the specified ID.
-
-    Args:
-        id: The ID of the item.
-
-    Returns:
-        item: The saved item, or None if no items match the specified ID.
-    """
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
-
-
-def add_item(title):
-    """
-    Adds a new item with the specified title to the session.
-
-    Args:
-        title: The title of the item.
-
-    Returns:
-        item: The saved item.
-    """
-    items = get_items()
-
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
-
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
+    
 
-    # Add the item to the list
-    items.append(item)
-    session['items'] = items
+    cards = []
+    trellolists = json.loads(response.text)
```

Comment:
Nothing wrong with this, but worth noting that the response object has a helpful method for this;
```suggestion
    trellolists = response.json()
```